nav2_controller/src/controller_server.cpp

```diff
@@ -659,18 +659,32 @@ void ControllerServer::computeAndPublishVelocity()
     }
   }
 
+  RCLCPP_DEBUG(get_logger(), "Publishing velocity at time %.2f", now().seconds());
+  publishVelocity(cmd_vel_2d);
+
+  // Find the closest pose to current pose on global path
+  geometry_msgs::msg::PoseStamped robot_pose_in_path_frame;
+  rclcpp::Duration tolerance(rclcpp::Duration::from_seconds(costmap_ros_->getTransformTolerance()));
+  if (!nav_2d_utils::transformPose(
+          costmap_ros_->getTfBuffer(), current_path_.header.frame_id, pose,
+          robot_pose_in_path_frame, tolerance))
+  {
+    RCLCPP_ERROR(get_logger(), "Robot pose is not available.");
```

Comment:
I think an exception should be thrown, this is a true error condition if TF transformations are not available

Reply:
done